src/arrayMethodSort.js

```diff
@@ -3,10 +3,42 @@
 /**
  * Implement method Sort
  */
+
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
+  [].__proto__.sort2 = function(compareFunction = compareAsStrings) {
     // write code here
+    let count;
+
+    do {
+      count = 0;
+
+      for (let i = 1; i < this.length; i++) {
+        const prevValue = this[i - 1];
+        const currentValue = this[i];
+
+        if (compareFunction(prevValue, currentValue) > 0) {
+          count++;
+          this[i - 1] = currentValue;
+          this[i] = prevValue;
+        }
+      }
+    } while (count > 0);
+
+    return this;
   };
 }
 
+const compareAsStrings = (a, b) => {
+  const stringA = String(a);
+  const stringB = String(b);
+
+  if (stringA > stringB) {
+    return 1;
+  } else if (stringA === stringB) {
+    return 0;
+  } else {
+    return -1;
+  }
+};
```

Comment:
you don't need `if...else` construction here, you can avoid it where possible.
We had this on the checklist in some of our previous tasks